buildres/linux/native-messaging-host/chromium/org.jabref.jabref.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "org.jabref.jabref",
+  "description": "JabRef",
+  "path": "/opt/jabref/lib/jabrefHost.py",
+  "type": "stdio",
+  "allowed_origins": [
+    "chrome-extension://hbngbnhgegoecdhfmndifldiaonjdgbn/"
```

Comment:
I guess the key needs to be changed as soon as the extension is published to the chorme store, right?

Reply:
I've never submitted a chrome extension, but I think that this key will work for the tests (web-ext run), and we could add another one for the "release" version..
but I might be wrong and the "published" key could work for the unreleased version as well, I'm not sure about that..

Reply:
Ok, I'll try to publish a (preliminary) version to the Chrome store and then update the ID.

Reply:
The firefox extension needs updating as well, to reflect the name change (and thus the id change)

Reply:
Chrome key: bifehkofibaamoeaopjglfkddgkijdlh

I cannot update the Firefox addon right now, because then it does no longer work with JabRef 4.3.1. Thus, this has to wait until 5.0 is released.